firecracker/microvm.py

```diff
@@ -11,6 +11,7 @@
 from tempfile import NamedTemporaryFile
 from typing import Optional, Tuple, List
 
+from network.network import Network
```

Comment:
Use a relative import when importing from the local project, else it could collide or be confused with a third party library.
```suggestion
from ..network.network import Network
```

Reply:
PyCharm automatically renamed it to that when I moved the file. And as I saw it being used like this in firecracker_microvm.py for other stuff, I left it as is. But I'll change it.